weave/ops_arrow/dict.py

```diff
@@ -111,6 +111,47 @@ def pick(self, key):
     return ArrowWeaveList(result, object_type, self._artifact)
 
 
+def _ensure_awl_for_merge(
+    ensure_target: typing.Union[dict, ArrowWeaveList],
+    compare_target: typing.Union[dict, ArrowWeaveList],
+) -> ArrowWeaveList:
+    if not isinstance(ensure_target, ArrowWeaveList):
+        assert isinstance(compare_target, ArrowWeaveList)
+        utility_awl = convert.to_arrow([ensure_target])
+        return ArrowWeaveList(
+            pa.repeat(ensure_target, len(compare_target)),
```

Comment:
is `repeat` fast?

Reply:
Yep. We use it in other places in the codebase to achieve something similar